docs/Managing-Indexes.md

```diff
@@ -19,6 +19,7 @@ Fill out the search index form, populating the fields with your custom values.
 - Channel Name - the index will only be triggered by web page item creation or modication in the selected website channel
 - Indexing Strategy - the indexing strategy specified in code during dependency registration of a custom indexing strategies.
   - If you want the default strategy to appear here, register it explicitly in `IServiceCollection.AddKenticoLucene()` method
+- Lucene Analyzer - the Lucene analyzer which indexes use to analyze text.
```

Comment:
An index owns an analyzer, not the opposite. An analyzer will be referenced by an index and when the index is rebuilt this analyzer is used to analyze text by selected strategy. When a service calls an index it can use the analyzer to analyze text which will be comapred to the indexed items. The index owns the analyzer and when some other member wants to use the analyzer it must request it from the index.